client/lib/analytics/ad-tracking.js

```diff
@@ -787,13 +832,27 @@ function recordProduct( product, orderId ) {
 		// Google AdWords
 		if ( isAdwordsEnabled ) {
 			if ( window.google_trackConversion ) {
+				let googleConversionId, googleConversionLabel;
+
+				if ( product.is_free_signup ) {
+					if ( isJetpackPlan ) {
+						googleConversionId = TRACKING_IDS.googleFreeSignupConversionIdJetpack;
+						googleConversionLabel = TRACKING_IDS.googleFreeSignupConversionLabelJetpack;
+					} else {
+						googleConversionId = TRACKING_IDS.googleFreeSignupConversionId;
+						googleConversionLabel = TRACKING_IDS.googleFreeSignupConversionLabel;
+					}
+				} else if ( isJetpackPlan ) {
+					googleConversionId = ADWORDS_CONVERSION_ID_JETPACK;
```

Comment:
It would be nice to have ADWORDS_CONVERSION_ID_JETPACK and ADWORDS_CONVERSION_ID declared in the same way as the new constants but we can leave that for another day :)